Cargo.toml

```diff
@@ -61,6 +61,8 @@ integer-sqrt = "0.1.3"
 slog = { version = "2.5.2", features = [ "max_level_trace" ] }
 slog-term = "2.6.0"
 slog-json = { version = "2.3.0", optional = true }
+chrono = "0.4.19"
+libc = "0.2.82"
```

Comment:
This is already a transitive dependency, right?

Reply:
Correct